invenio_sequencegenerator/models.py

```diff
@@ -63,13 +66,12 @@ def validate_meta_template(self, key, value):
         """Validate template string of template definition."""
         if not self.COUNTER_REGEX.search(value):
             raise InvalidTemplate('No counter placeholder')
-        value = double_counter(value, self.COUNTER_REGEX)
         return value
 
-    @hybrid_property
+    @property
```

Comment:
From what I understood, if it was a `@hybrid_property`, it would require translation to an SQL expression when used in a query or sth like that. but it uses the `undouble_counter` method that uses regex and this cannot be translated (at least automatically) to SQL.